Adyen/Core/AdyenContext/AdyenContext.swift

```diff
@@ -19,16 +19,20 @@ public final class AdyenContext: PaymentAware {
     public private(set) var payment: Payment?
 
     @_spi(AdyenInternal)
-    public let analyticsProvider: AnalyticsProviderProtocol
-
+    public var analyticsProvider: AnalyticsProviderProtocol
```

Comment:
Is this a good idea?
Maybe better to modify it via `AnalyticsConfiguration` ?

Reply:
Please check https://github.com/Adyen/adyen-ios/pull/1473 for an alternative implementation